enterprise/backend/src/metabase_enterprise/audit_app/pages/common.clj

```diff
@@ -4,12 +4,12 @@
    [clojure.core.async :as a]
    [clojure.core.memoize :as memoize]
    [clojure.walk :as walk]
-   [honeysql.core :as hsql]
-   [honeysql.format :as hformat]
-   [honeysql.helpers :as hh]
+   [honey.sql :as sql]
```

Comment:
I would expect an `h` in honeysql namespace aliases.

Reply:
Something like `hsql` conflicts with the aliases we're using for Honey SQL 1, and the suggested alias in the Honey SQL 2 documentation is `sql` -- https://github.com/seancorfield/honeysql#usage -- so I thought it would work here. I'm open to other suggestions tho that don't conflict with our Honey SQL 1 aliases

Reply:
I can get used to it. 👍